packages/store/gas-report.txt

```diff
@@ -0,0 +1,566 @@
+[
```

Comment:
I assume this is a leftover from before gas-report got changed, you should delete this file, it's `.json` now
(same for the one in `world`)